HelloKelly/HelloKelly/Program.cs

```diff
@@ -8,15 +8,21 @@ namespace HelloKelly
 {
     class Program
     {
+        
         static void Main(string[] args)
         {
             string userName;
             Console.WriteLine("What is your name?");
             userName=Console.ReadLine();
-            Console.WriteLine("Nice to meet you, " + userName + "!");
+            Greeting(userName);
             Console.Write("Press <Enter> to exit... ");
             while (Console.ReadKey().Key != ConsoleKey.Enter) { }
 
         }
+        static void Greeting(string userName )
+        {
+            Console.WriteLine("Nice to meet you, " + userName + "!");
```

Comment:
Try this fun new thing where you add a $ before the first quote in the string. Then, you can replace (" + userName + ") with just {userName}.